config/jobs/periodic/kubernetes/perf-tests/perf-tests-kubernetes-periodics.yaml

```diff
@@ -126,7 +127,8 @@ periodics:
         export PATH=$GOPATH/bin:$PATH
         export GO111MODULE=on
 
-        make install-all
+        make install
+        make install-tester-exec
```

Comment:
```suggestion
        make install install-tester-exec
```

Reply:
Had two minds for this as the following command also had the `install` prefix. Addressed! 
Thanks @mkumatag.